pkg/metrics/metrics.go

```diff
@@ -245,21 +246,30 @@ func (c *Config) Configure(ctx context.Context) error {
 	return nil
 }
 
-// validate will verify if cluster monitoring is enabled in the operator namespace.
-// If the label is not present, it will log and send warning events to the user.
+// validate will verify if cluster monitoring is enabled in the operator namespace. If the label is set false or not
+// present, it will log and send warning events to the user. If the label hold a non-boolean value, returns error.
 func (c *Config) validate(ctx context.Context) (bool, error) {
 	// validate if metrics label is added to namespace
 	wmcoNamespace, err := c.CoreV1().Namespaces().Get(ctx, c.namespace, metav1.GetOptions{})
 	if err != nil {
 		return false, errors.Wrap(err, "error getting operator namespace")
 	}
-	if wmcoNamespace.Labels["openshift.io/cluster-monitoring"] != "true" {
-		metricsEnabled = false
+
+	boolVal := false
+	value, ok := wmcoNamespace.Labels["openshift.io/cluster-monitoring"]
+	// if the label exists, update value from default of false
+	if ok {
```

Comment:
This can be combined:
```suggestion
if value, ok := wmcoNamespace.Labels["openshift.io/cluster-monitoring"]; ok {
```